eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -129,6 +129,9 @@ jobs:
         OSVmImage: "windows-2019"
         TestTargetFramework: net461
         AdditionalTestArguments: /p:UseProjectReferenceToAzureClients=true
+      - DisplayName: "Test on Windows for .NET 5.0"
```

Comment:
Is there a reason you picked windows?

Reply:
I don't have a good reason why any other platform would be better.

Reply:
Storage tests on Linux are significantly slower. That's +1 for windows (and MacOS).

Reply:
linux and macOS are generally faster, though. That said, net5.0 is supposed to bring a lot of perf improvements to .NET in general, so they may still finish before net461 or even netcoreapp2.1.

Reply:
It would be nice, as a goal, to have tests on multiple platforms for our most used target framework